diesel/src/pg/expression/array_comparison.rs

```diff
@@ -94,6 +81,33 @@ where
 
 impl_selectable_expression!(Any<Expr>);
 
+/// Creates a PostgreSQL `ALL` expression.
+///
+/// As with most bare functions, this is not exported by default. You can import
+/// it specifically as `diesel::pg::expression::dsl::all`, or `diesel::dsl::all`.
+///
+/// # Example
+///
+/// ```rust
+/// # include!("../../doctest_setup.rs");
+/// # use diesel::dsl::*;
+/// #
+/// # fn main() {
+/// #     use schema::users::dsl::*;
+/// #     let connection = establish_connection();
+/// #     connection.execute("INSERT INTO users (name) VALUES ('Jim')").unwrap();
```

Comment:
These comments are not displayed in the documentation, so I think it is not necessary.

@weiznich 
Is there any particular reason why they are commented out?